prov/efa/src/efa_cq.c

```diff
@@ -50,42 +49,53 @@ static uint64_t efa_cq_wc_to_fi_flags(struct efa_wc *wc)
 	}
 }
 
+/* @brief Unlike ibv_poll_cq, wide completion APIs do not write to ibv_wc struct. We need to do this manually.
+ * This is inspired by rdma-core, i.e. `efa_process_cqe` and `efa_process_ex_cqe`.
```

Comment:
Can you also add description for argument and return value?

Reply:
Sure. Will address.